vulnerable_people_form/form_pages/address_lookup.py

```diff
@@ -59,7 +59,7 @@ def get_address_lookup():
 
 def _create_test_address(postcode):
     return [{
-        "text": f"{current_app.config['POSTCODE_TIER_OVERRIDE'][postcode]}, Test Lane, City, {postcode}",
+        "text": f"{current_app.postcode_tier_override[postcode]}, Test Lane, City, {postcode}",
```

Comment:
`building_and_street_line_1:` needs to be updated to use `current_app.postcode_tier_override[postcode]` too